src/runtime/edbml/scripts/ts.ui.ToolBarSpirit.edbml

```diff
@@ -23,7 +23,11 @@
 		toolbar.checkbox,
 		toolbar.$allactions()
 	);
+
+	renderpagerstatus(id, toolbar.pager);
```

Comment:
Did you see the `renderLeft` above ? It will render different stuff depends on different condition. You can't just render another thing there. it would break a lot of stuff.

Reply:
Yes, we could put it there as well, this way seemed to me a little cleaner. What do you think will break? This is only on pager, the pager bar will not have other things set.

Reply:
Updated with code moved in `renderLeft`/`renderlabels`. What do you think?